### PR TITLE
allow for identity POST_TRANSFORM

### PR DESCRIPTION
**Pull request by jppgks**
In https://github.com/ludwig-ai/ludwig, we require access to the raw logits since we already have our own "post-transform" logic. With this PR, a user can predict raw logits by using the following `extra_config`:

```python
from hummingbird.ml import convert
from hummingbird.ml.operator_converters import constants

lgbm_model = ...
convert(lgbm_model, "torch", extra_config={constants.POST_TRANSFORM: None})
```

### CODE REVIEW

**Comment by jppgks** [comment]
@ksaur would you have bandwidth to review this small PR? Many thanks!

**Review by ksaur — CHANGES_REQUESTED** [review]
Thanks for the contribution! 
Can you please add a test case that demonstrates use?

**Comment by jppgks** [comment]
@microsoft-github-policy-service agree company="Predibase, Inc."

**Comment by jppgks** [comment]
@ksaur Thanks for reviewing! I just added a test case and agreed to the CLA.

**Review by ksaur — APPROVED** [review]
thanks!
